src/txdb.h

```diff
@@ -22,6 +22,34 @@ class CBlockIndex;
 class CCoinsViewDBCursor;
 class uint256;
 
+/*
+struct to store and retrieve xfield aggpubkey change information into level db blocks db
+*/ 
+
+struct XFieldEntry {
```

Comment:
PR #234 contains the better classes that can manage multiple xfields. After this PR I'll try to make those methods templates so that we need not repeat writing similar code for each type.